app/assets/stylesheets/views/_history-people.scss

```diff
@@ -1,20 +1,22 @@
 .historic-people-index {
+  $gutter: 30px;
```

Comment:
We use the [govuk-spacing scale](https://govuk-design-system-preview.netlify.app/styles/spacing/) now, so we should be able to remove this and replace all instances of `$gutter` (I've commented in this PR where I've spotted usages, but may have missed some)